src/pybel/struct/__init__.py

```diff
@@ -21,11 +21,11 @@
 from .summary import *
 
 __all__ = (
-        graph.__all__ +
```

Comment:
huh. for some reason, pycharm adds this extra tab. At some point, I finally gave up trying to fight it @scolby33 what do you think?

Reply:
I'm afraid I don't have an answer for you. What does Black do? Is it worth it to try to change the pycharm configuration? Either way *looks* fine to me.